src/deepsparse/loggers/helpers.py

```diff
@@ -62,14 +62,16 @@ def unwrap_logs_dictionary(
     :param seperator: The seperator to use when composing the parent and child
         identifiers
     :return: A generator that:
-        - if `value` is not a dictionary:
+        - if `value` is a dictionary:
```

Comment:
might want to rebrand this function in general to `unwrap_logged_value` since we also do special list logic.